.golangci.toml

```diff
@@ -77,6 +77,10 @@
   max-per-linter = 0
   max-same-issues = 0
   exclude = [
-    "Error return value of .((os\\.)?std(out|err)\\..*|.*Close|.*Flush|os\\.Remove(All)?|.*printf?|os\\.(Un)?Setenv). is not checked",
-    "should have a package comment, unless it's in another file for this package",
+    "Error return value of .((os\\.)?std(out|err)\\..*|.*Close|.*Flush|os\\.Remove(All)?|.*printf?|os\\.(Un)?Setenv|\\(k8s\\.io/client-go/tools/cache\\.SharedInformer\\)\\.AddEventHandler). is not checked",
+    "package-comments: should have a package comment",
+    "import '[^']+' is not allowed from list 'Main'",
+    "if-return: redundant if ...; err != nil check, just return error instead.",
+    "unused-parameter: parameter '[^']+' seems to be unused, consider removing or renaming it as _",
+    "tag is not aligned, should be: description:",
```

Comment:
you should revert this modification.
And add the following snippet inside the `.golangci.toml` file:

```
[linters-settings.depguard]
    [linters-settings.depguard.rules]
      [linters-settings.depguard.rules.main]
        [[linters-settings.depguard.rules.main.deny]]
        pkg = "github.com/instana/testify"
        desc = "not allowed"

        [[linters-settings.depguard.rules.main.deny]]
        pkg = "github.com/pkg/errors"
        desc = "Should be replaced by standard lib errors package"
```

The other problem should be fixed (and not ignored) except `"package-comments: should have a package comment"`

Reply:
Will do except fixing the actual problems, you know your code better 😅

Reply:
Could fix linter issues except AddEventHandler and unused-parameter.

Reply:
Sorry can you revert the change related to tagalign and add this configuration:
```toml
[linters-settings.tagalign]
    align = false
    sort = true
    order = [
        "description",
        "json",
        "toml",
        "yaml",
        "yml",
        "label",
        "label-slice-as-struct",
        "file",
        "kv",
        "export"
    ]
```

Reply:
You can fix `unused-parameter`: you just have to replace the name of the parameter with `_`.

And `AddEventHandler` can also be fixed but it's a bit more complex, so you can ignore them inside this PR but I will fix them after the merge of your PR.

Reply:
Already tried that yesterday, it doesn't work, just another linter pops up instead of revive:
```File is not `gci`-ed with --skip-generated -s standard -s default (gci)```
It would be much easier if you fix those special cases yourself, you know your code and Go better than me.

And no single check has been run yet, can't you start the GitHub Actions manually to see if all integration tests work?

Reply:
It works I did it locally.